3 - PB/Documentazione esterna/Manuale Utente/Manuale Utente.typ

```diff
@@ -197,8 +197,31 @@ Ruotando la rotella del mouse in avanti è possibile avvicinarsi all'oggetto des
 
 Premendo il tasto destro del mouse e spostando il mouse in una delle quattro direzioni (su, giù, destra e sinistra) otterremo il panning della camera, ovvero lo spostamento sui due assi della camera rispetto al piano come mostrato in figura @camera_panning.
 
+=== Movimento con frecce direzionali
+
+#figure(
+    grid(
+        columns: 2,
+        rows:    (auto, auto),
+        [ #image("./imgs/keyboard_arrow.png", width: 40%)],
+        [ #image("./imgs/keyboard_shift.jpg", width: 40%)],
+    ),caption: [Movimento con le frecce direzionali],
+) <camera_direzionali>
+
+L' ultimo metodo per muoversi all'interno dell'ambiente è tramite l'uso delle frecce direzionali, che permettono il movimento nella direzione indicata dalla freccia. Con il tasto Shift è possibile aumentare la velocità di spostamento con le frecce direzionali.
```

Comment:
Aggiungerei "... l'uso delle frecce direzionali (o in alternativa i tasti W, A, S, D)" e toglierei alla fine "con le frecce direzionali"

Reply:
sai che credo non si possa usare wasd per lo spostamento, ma solo le freccie, controllo

Reply:
ho detto una cazzata, sistemo